client/state/data-layer/analytics/adTracking.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { recordSelectedVertical } from 'lib/analytics/ad-tracking';
+import { SIGNUP_STEPS_SURVEY_SET } from 'state/action-types';
+
+function handleSignupStepsSurveyStart( _, action, next ) {
```

Comment:
hm. we don't particularly use `_` for unused vars. I support this because `_` isn't special and we can't ignore multiple vars by reusing `_`